pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/httproute_reconciler.go

```diff
@@ -595,38 +595,57 @@ func (r *RawHTTPRouteReconciler) reconcileTransformerHTTPRoute(ctx context.Conte
 	if err != nil {
 		return err
 	}
+
+	// reconcile httpRoute
+	httpRouteName := constants.TransformerServiceName(isvc.Name)
```

Comment:
Wouldn't it be better to use `CheckResultType` for comparison and subsequent actions during reconciliation? Similar to how the `deployment_reconciler`. Would keep the code more consistent

Reply:
let's create an issue so as to not lose track, it can be a good first issue for someone to get started with this repository in the future

Reply:
We have someone starting next week so perfect timing :)

Reply:
Opened JIRA